### PR TITLE
Layout all pages according design

### PR DESCRIPTION
**Pull request by BogdanParashchak**

### CODE REVIEW

File: src/component/Header/HeaderMobile/HeaderMobile.module.css [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
   display: none;
 }
 
 @media (max-width: 768px) {
-  .header__mobile{
-/*    height: 29px;
-    margin-top: 18px;*/
+  .header__mobile {
     height: 42px;
     margin-top: 25px;
```
Comment (Vahtangius) [inline src/component/Header/HeaderMobile/HeaderMobile.module.css]: Convert to em

File: src/component/VinylPages/OneVinylPage/VinylOfferList/VinylOfferItem/VinylOfferItem.module.css [hunk 2]
```diff
@@ -4,8 +4,10 @@
 }
 
 .shopList__itemImage {
-  flex: 0 0 10.83em;
-  height: 2.78em;
+/*  flex: 0 0 10.83em;
+  height: 2.78em;*/
```
Comment (Vahtangius) [inline src/component/VinylPages/OneVinylPage/VinylOfferList/VinylOfferItem/VinylOfferItem.module.css]: Delete comments

File: src/component/VinylPages/OneVinylPage/VinylOfferList/VinylOfferList.module.css [hunk 3]
```diff
@@ -82,9 +94,13 @@
     width: 100%;
   }
 
-/*  .shopList__itemDirection: last-child {
-    justify-content: flex-end;
-  }*/
+  .shopList__goToShopWidth {
+   justify-content: flex-end;
+  }
+
```
Comment (Vahtangius) [inline src/component/VinylPages/OneVinylPage/VinylOfferList/VinylOfferList.module.css]: Delete comment and Convert to em

**Review by Vahtangius — CHANGES_REQUESTED** [review]

File: src/component/Header/HeaderMobile/HeaderMobile.module.css (hunk 1 above)
Comment (BogdanParashchak) [reply src/component/Header/HeaderMobile/HeaderMobile.module.css]: done

File: src/component/VinylPages/OneVinylPage/VinylOfferList/VinylOfferItem/VinylOfferItem.module.css (hunk 2 above)
Comment (BogdanParashchak) [reply src/component/VinylPages/OneVinylPage/VinylOfferList/VinylOfferItem/VinylOfferItem.module.css]: done

File: src/component/VinylPages/OneVinylPage/VinylOfferList/VinylOfferItem/VinylOfferItem.module.css [hunk 4]
```diff
@@ -4,8 +4,10 @@
...
+  height: 50px;
+  width: 195px;
```
[the remark this answers is not in the document]
Comment (BogdanParashchak) [reply src/component/VinylPages/OneVinylPage/VinylOfferList/VinylOfferItem/VinylOfferItem.module.css:8]: done

File: src/component/VinylPages/OneVinylPage/VinylOfferList/VinylOfferItem/VinylOfferItem.module.css [hunk 5]
```diff
@@ -29,17 +31,22 @@
 .button {
   text-transform: uppercase;
   display: block;
-  font-size: 0.78em;
+/*  font-size: 0.78em;
   width: 10.21em;
   height: 2.72em;
-  line-height: 2.72;
```
[the remark this answers is not in the document]
Comment (BogdanParashchak) [reply src/component/VinylPages/OneVinylPage/VinylOfferList/VinylOfferItem/VinylOfferItem.module.css]: done

File: src/component/VinylPages/OneVinylPage/VinylOfferList/VinylOfferItem/VinylOfferItem.module.css [hunk 6]
```diff
@@ -29,17 +31,22 @@
...
+  line-height: 2.72;*/
+  width: 140px;
+  height: 40px;
+  line-height: 40px;
```
[the remark this answers is not in the document]
Comment (BogdanParashchak) [reply src/component/VinylPages/OneVinylPage/VinylOfferList/VinylOfferItem/VinylOfferItem.module.css:34]: done

File: src/component/VinylPages/OneVinylPage/VinylOfferList/VinylOfferItem/VinylOfferItem.module.css [hunk 7]
```diff
@@ -79,6 +86,4 @@
   .inStock {
     background-size: 15px 11.5px;
```
[the remark this answers is not in the document]
Comment (BogdanParashchak) [reply src/component/VinylPages/OneVinylPage/VinylOfferList/VinylOfferItem/VinylOfferItem.module.css:77]: done

File: src/component/VinylPages/OneVinylPage/VinylOfferList/VinylOfferList.module.css [hunk 8]
```diff
@@ -12,12 +12,17 @@
   justify-content: space-between;
   align-items: center;
   border-bottom: thin solid rgba(255, 255, 255, 0.5);
+  height: 80px;
```
[the remark this answers is not in the document]
Comment (BogdanParashchak) [reply src/component/VinylPages/OneVinylPage/VinylOfferList/VinylOfferList.module.css]: done

File: src/component/VinylPages/OneVinylPage/VinylOfferList/VinylOfferList.module.css [hunk 9]
```diff
@@ -30,19 +35,24 @@
 }
 
 .shopList__catalogWidth {
-  width: 9.88em;
+  /*width: 9.88em;*/
```
[the remark this answers is not in the document]
Comment (BogdanParashchak) [reply src/component/VinylPages/OneVinylPage/VinylOfferList/VinylOfferList.module.css]: done

File: src/component/VinylPages/OneVinylPage/VinylOfferList/VinylOfferList.module.css [hunk 10]
```diff
@@ -30,19 +35,24 @@
...
+  width: 125px;
```
[the remark this answers is not in the document]
Comment (BogdanParashchak) [reply src/component/VinylPages/OneVinylPage/VinylOfferList/VinylOfferList.module.css]: done

**Review by Vahtangius — APPROVED** [review]

**Pull request merged**